deploy-disconnected-registry/verify_olm_sync.sh

```diff
@@ -42,25 +21,9 @@ fi
 
 echo ">>>> Verifying OLM Sync: ${1}"
 registry_login ${DESTINATION_REGISTRY}
-for packagemanifest in $(oc --kubeconfig=${TGT_KUBECONFIG} get packagemanifest -n openshift-marketplace -o name ${PACKAGES_FORMATED}); do
-    for package in $(oc --kubeconfig=${TGT_KUBECONFIG} get ${packagemanifest} -o jsonpath='{.status.channels[*].currentCSVDesc.relatedImages}' | sed "s/ /\n/g" | tr -d '[],' | sed 's/"/ /g'); do
-        echo "Verify Package: ${package}"
-        #if next command fails, it means that the image is not already in the destination registry, so output command will be error (>0)
-        skopeo inspect docker://"${DESTINATION_REGISTRY}"/"${OLM_DESTINATION_REGISTRY_IMAGE_NS}"/$(echo ${package} | awk -F'/' '{print $2}')-$(basename ${package}) --authfile "${PULL_SECRET}"
-    done
-done
-
-echo ">>>> Verifying Certified OLM Sync: ${1}"
-registry_login ${DESTINATION_REGISTRY}
-for packagemanifest in $(oc --kubeconfig=${TGT_KUBECONFIG} get packagemanifest -n openshift-marketplace -o name ${CERTIFIED_PACKAGES_FORMATED}); do
-    for package in $(oc --kubeconfig=${TGT_KUBECONFIG} get ${packagemanifest} -o jsonpath='{.status.channels[*].currentCSVDesc.relatedImages}' | sed "s/ /\n/g" | tr -d '[],' | sed 's/"/ /g'); do
-        echo "Verify Package: ${package}"
-        #if next command fails, it means that the image is not already in the destination registry, so output command will be error (>0)
-        skopeo inspect docker://"${DESTINATION_REGISTRY}"/"${OLM_DESTINATION_REGISTRY_IMAGE_NS}"/$(echo ${package} | awk -F'/' '{print $2}')-$(basename ${package}) --authfile "${PULL_SECRET}"
-    done
-done
-#In this case, we don't need to mirror catalogs, everything is already there
-
-recover_mapping
 
-exit 0
+# Always trigger the mirror, let's let oc-mirror
+# decide whether images need to be refreshed or not.
+# TODO(flaper87): Add check to verify whether the registry
+# needs to be installed or not.
+exit 1
```

Comment:
Seems like we don't want to use this script.
Isn't it better to just rename it to verify_olm_sync.sh_disabled ?

Reply:
mmh, it still does the registry login and I think we should verify, which serves as a verification of whether the registry has been installed. I didn't remove it because I do want to add some validations here.